alpha/apps/kaltura/lib/myEntryUtils.class.php

```diff
@@ -2263,7 +2263,7 @@ public static function getVolumeMapContent($flavorAsset, $mapScale = null, $dura
 		$content = null;
 
 		$cacheStore = kCacheManager::getSingleLayerCache(kCacheManager::CACHE_TYPE_VOLUME_MAP);
-		$cacheKey = $entryId . '_volumeMap';
+		$cacheKey = $entryId . '_volumeMap' . $mapScale;
```

Comment:
I think you should also include the version of the asset in your cache key.
In case someone replaces content based on what I see the code will return the old volume map.